tests/providers/cncf/kubernetes/operators/test_kubernetes_pod.py

```diff
@@ -426,3 +426,40 @@ def test_node_selector(self):
         client = ApiClient()
         assert isinstance(result.spec.node_selector, dict)
         assert client.sanitize_for_serialization(result)['spec']['nodeSelector'] == node_selector
+
+    def test_hostpid_and_hostipc(self):
```

Comment:
i think it's better to parameterize a test like this.  will make it easier to see exactly what you're testing, and reduce some duplication also.